glossary.md

```diff
@@ -1,16 +1,80 @@
 You may also need to check https://chromium.googlesource.com/chromiumos/docs/+/master/glossary.md.
 
-* LGTM: "Looks good to me", commonly used to approve a code review.
-* PTAL: Please take a look.
-* RSLGTM: "Rubber-stamp looks good to me". The reviewer approving without doing
+* ABI: [Application Binary Interface](https://en.wikipedia.org/wiki/Application_binary_interface)
+* ASAP: As Soon As Possible.
+* ASYNC: Asynchronous
+* BE: Big Endian - Byte order style in a multibyte data.
+  (Opposite of LE, Little Endian)
+* CI: Continuous Integration.
+* CITGM: Canary In The Gold Mine - a smoke test unit in the repo that
+  contain popular npm modules.
+* CJS: Common JS.
+* CLDR: Common Locale Data Repository.
+* CLI: [Command Line Interface](https://en.wikipedia.org/wiki/Command-line_interface)
+* CMD: Command.
+* CVE: Common Vulnerebilities and Exposures - A database that
+  maintains reported security exposures.
+* ECMA: European Computer Manufacturers Association - A body that
+  governs JS spec among other things.
+* ENV: Environment - General term for execution environment.
+* EOF: [End Of File](https://en.wikipedia.org/wiki/End-of-file)
+* EOL: End Of Line (when used within a program), End of Life
+  (when used within project documents).
+* ESM: ECMA Script Module.
+* ETW: Event Tracing for Windows.
+* FD: File Descriptor.
+* FFDC: First Failure Data Capture - Common terms for logs, traces
+  and dumps that are produced by default on program error.
+* FIPS:  Federal Information Processing Standards.
+* FS: File System.
+* ICU: International Components for Unicode.
+* IPC: Inter Process Communication.
+* JIT: Just In Time - General term for dynamic compiler in
+  managed runtimes.
+* JS/C++ boundary: A layer that bridges between JS APIs and the C++
```

Comment:
I don't think is accurate - when we talk about `JS/C++ boundary` we are usually referring to the boundary between V8's runtime and the JS code execution which are commonly crossed when invoking JS functions with C++ linkage (e.g. interceptors), i.e. something like this

```
    frame #11: 0x00000001009703db node`Builtins_InterpreterEntryTrampoline + 187
    frame #12: 0x00000001009703db node`Builtins_InterpreterEntryTrampoline + 187
    frame #13: 0x00000001009703db node`Builtins_InterpreterEntryTrampoline + 187
    frame #14: 0x000000010096dfda node`Builtins_JSEntryTrampoline + 90
    frame #15: 0x000000010096ddb8 node`Builtins_JSEntry + 120
```

Instead of the conceptual layer in the implementation that abstracts platform capabilities. The boundary could very well exist when we are reaching into the C++ to do something that has nothing to do with the platform, e.g. URL parsing.

Reply:
Thanks, Changed it. PTAL.